Ginger/GingerCoreNET/RunLib/CLILib/CLIArgs.cs

```diff
@@ -79,17 +79,21 @@ public string CreateContent(Solution solution, RunsetExecutor runsetExecutor, CL
             return args;
         }
 
-        public void Execute(RunsetExecutor runsetExecutor)
+        public void LoadGeneralConfigurations(string content, CLIHelper cliHelper)
         {
-            runsetExecutor.RunRunset();
+
         }
-       
 
-        public void LoadContent(string args, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
-        {      
-            
+        public void LoadRunsetConfigurations(string content, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)